### PR TITLE
Fixed pytest run on Tests and Utils

### PR DESCRIPTION
**Pull request by jochman**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://github.com/demisto/etc/issues/29435

## Description
Refactored names to not be collected by Pytest, added import to globally work (slack import).
Running pytest on both Utils and Tests folder.

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Tests

### CODE REVIEW

File: Tests/test_integration.py [hunk 1]
```diff
@@ -818,6 +820,7 @@ def configure_proxy_unsecure(integration_params):
 # 3. wait for playbook to finish run
 # 4. if test pass - delete incident & instance
 # return playbook status
+@pytest.mark.skip(reason="Not a test")
 def test_integration(client, server_url, integrations, playbook_id, prints_manager, options=None, is_mock_run=False,
```
Comment (Itay4) [inline Tests/test_integration.py:824]: why not rename the func like you did in the classes?

File: Tests/test_content.py [hunk 2]
```diff
@@ -23,7 +23,10 @@
 import requests
 import demisto_client.demisto_api
 from demisto_client.demisto_api.rest import ApiException
-from slackclient import SlackClient
+try:
```
Comment (Itay4) [inline Tests/test_content.py:26]: why is this needed?

File: dev-requirements-py3.txt [hunk 3]
```diff
@@ -20,4 +20,6 @@ circleci==1.2.2
 pylint==2.6.0
 coloredlogs==14.0
 pandas==1.1.0
-pyarrow==1.0.0
\ No newline at end of file
+pyarrow==1.0.0
+prettytable==1.0.1
```
Comment (Itay4) [inline dev-requirements-py3.txt:24]: why are those needed? we have them in the build reqs

File: Tests/test_content.py (hunk 2 above)
Comment (jochman) [reply Tests/test_content.py:26]: We have an old slack in our dockerimage.
On local envs mostly we will get the new slack.

File: dev-requirements-py3.txt (hunk 3 above)
Comment (jochman) [reply dev-requirements-py3.txt:24]: When the reqs are in the requirements, it'll assure everyone will get them

File: Tests/test_content.py (hunk 2 above)
Comment (Itay4) [reply Tests/test_content.py:26]: please comment why this is needed

File: dev-requirements-py3.txt (hunk 3 above)
Comment (Itay4) [reply dev-requirements-py3.txt:24]: please remove

**Review by Itay4 — APPROVED** [review]
